tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -118,13 +118,6 @@ fi
 cd /tracim/backend/
 tracimcli search index-create -c /etc/tracim/development.ini
 
-if [ "$UPDATE_INDEX_ELASTICSEARCH" = "1" ]; then
```

Comment:
Why is this code is dropped ?

Reply:
We drop this feature for the moment because starting tracim with populate take too much time.
We added documentation to make this update manually directly in the container.

Reply:
ok.